dom.bs

```diff
@@ -1270,7 +1270,7 @@ for discussion).
 <ol>
  <li><p>If <var>event</var>'s <a>stop propagation flag</a> is set, then return.
 
- <li><p>Let <var>listeners</var> be a new <a>list</a>.
+ <li><p>Let <var>listeners</var> be a new <a for=/>list</a>.
```

Comment:
I don't like link defaults. What was that other option we thought of enabling at some point for WHATWG?